posthog/api/test/test_hog_function.py

```diff
@@ -1441,3 +1442,85 @@ def test_list_hog_functions_ordered_by_execution_order_and_created_at(self):
             "Function 3",  # execution_order=2
             "Function 4",  # execution_order=null
         ]
+
+    @override_settings(HOG_TRANSFORMATIONS_CUSTOM_ENABLED_TEAMS=["2"])
+    def test_transformation_code_editing_restricted_by_team(self):
+        # Create team with ID 2
+        team_2 = Team.objects.create(id=2, organization=self.organization, name="Team 2")
```

Comment:
This is fine but for the future you could have done
```suggestion
        with self.settings(HOG_TRANSFORMATIONS_CUSTOM_ENABLED_TEAMS=[str(self.team.id)]:
```

instead of the override decorator